### DISCUSSION TITLE
Cannot open and use Table to collection anymore

### DISCUSSION THREAD

**Discussion by Davidxcl** [AntlerVC/firetable#1513] [Support / Q&A] [answered]
<img width="1041" alt="Screenshot 2023-12-19 at 17 56 45" src="https://github.com/rowyio/rowy/assets/111456376/e1dc83d7-485a-4b3a-9d8a-9b0317bf54c6">

I can no longer access my data, extensions, derivatives and so on in this collection, which is really bad. How can I contact support or see if this is being worked on?

I get this error every time I try to open my table from my events collection. I get this error in the console:

`RangeError: Invalid time value
    at Date.toISOString (<anonymous>)
    at FI (index-eb716aa4.js:2612:4887)
    at index-eb716aa4.js:3672:5369
    at dE (index-eb716aa4.js:3672:6010)
    at $qt (index-eb716aa4.js:3672:4352)
    at index-eb716aa4.js:3702:1428
    at jP._parse (index-eb716aa4.js:3702:1486)
    at jP._apply (index-eb716aa4.js:3702:913)
    at I3 (index-eb716aa4.js:3702:723)
    at eAi (index-eb716aa4.js:9862:23360)`

The URL is set to: "https://rowy.app/p/nightliveapp-de/table/events?filter=%255B%257B%2522key%2522%253A%2522startsAt%2522%252C%2522operator%2522%253A%2522time-minute-equal%2522%252C%2522value%2522%253A%257B%2522seconds%2522%253A1703372400%252C%2522nanoseconds%2522%253A0%257D%252C%2522id%2522%253A%2522rNUs0UM8os8gBfYDjdVy%2522%257D%255D"
    
So I think this has something to do with the filters I used to filter my table, but I have no way of resetting the filters.

**Comment by Chadha93** [comment, marked as answer]
Hi, @Davidxcl It is an issue with the corrupted data in Firestore. You can visit the Firestore console https://console.firebase.google.com/ and under it, you should see two collections _rowy_ and <your-collection> you can check documents for both and update/remove the time value that is causing the issue. If this issue occurs after setting filters, you can delete filters. We'll be working on enhancing this specific error handler. 
![console](https://github.com/rowyio/rowy/assets/65453826/f1d80dc1-2f00-4361-9f79-ec11296b4e55)


You can join our Discord - https://www.rowy.io/discord and also connect with support via the app support form.

**Reply by Davidxcl** [reply]
Many thanks for the incredibly quick help! I have found the date in my Firestore. Can I simply delete the entire object in the "filters" array? 
![image](https://github.com/rowyio/rowy/assets/111456376/c535963e-6633-41b1-b129-c610a7b426ac)

**Reply by Chadha93** [reply]
Yes

**Reply by Davidxcl** [reply]
It worked 🙏  Thank you!
